### PR TITLE
Phase2-hgx327B Add a test suit to investigate why some of the SimHits are declared invalid in Geometry/HGCalCommonData

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add a test suit to investigate why some of the SimHits are declared invalid in Geometry/HGCalCommonData

#### PR validation:

Verified the test code and the scripts

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HGCalCommonData/test/python/testHGCalValidity_cfg.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                  VarParsing.VarParsing.varType.string,
+                  "geometry of operations: D88, D92")
+
+### get and parse the command line arguments
+options.parseArguments()
+print(options)
+
+####################################################################
+# Use the options
+from Configuration.Eras.Era_Phase2C11M9_cff import Phase2C11M9
```
Comment (srimanob) [inline Geometry/HGCalCommonData/test/python/testHGCalValidity_cfg.py:27]: Why is Phase2C11M9 used here? Should it be Phase2C17I13M9 for D88 and D92?

**Comment by srimanob** [comment]
+Upgrade

**Comment by bsunanda** [comment]
+geometry

**Comment by perrotta** [comment]
+1

**Pull request merged**